modules/desktop/test/tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
can we try "extends": "../tsconfig.json"
mabye we can reduce some redundant config